Corona-Warn-App/src/main/java/de/rki/coronawarnapp/risk/storage/legacy/RiskLevelResultMigrator.kt

```diff
@@ -77,18 +76,23 @@ class RiskLevelResultMigrator @Inject constructor(
     }
 
     companion object {
+
         @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
-        internal fun mapRiskLevelConstant(value: Int): RiskLevel {
-            return when (value) {
-                RiskLevelConstants.NO_CALCULATION_POSSIBLE_TRACING_OFF -> RiskLevel.NO_CALCULATION_POSSIBLE_TRACING_OFF
-                RiskLevelConstants.LOW_LEVEL_RISK -> RiskLevel.LOW_LEVEL_RISK
-                RiskLevelConstants.INCREASED_RISK -> RiskLevel.INCREASED_RISK
-                RiskLevelConstants.UNKNOWN_RISK_OUTDATED_RESULTS -> RiskLevel.UNKNOWN_RISK_OUTDATED_RESULTS
-                RiskLevelConstants.UNKNOWN_RISK_OUTDATED_RESULTS_MANUAL -> {
-                    RiskLevel.UNKNOWN_RISK_OUTDATED_RESULTS_MANUAL
-                }
-                else -> RiskLevel.UNDETERMINED
-            }
+        internal fun mapRiskLevelConstant(value: Int): RiskState = when (value) {
+            MigrationRiskLevelConstants.LOW_LEVEL_RISK -> RiskState.LOW_LEVEL_RISK
```

Comment:
Maybe just use LOW_RISK (instead of low level risk) for consistency as the other state is INCREASED_RISK